src/main/java/com/cs98/VerbatimBackend/repository/UserGroupRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.cs98.VerbatimBackend.repository;
+
+import com.cs98.VerbatimBackend.model.User;
+import com.cs98.VerbatimBackend.model.UserGroup;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserGroupRepository extends JpaRepository<UserGroup, Integer> {
+
+    UserGroup findByName(String name);
```

Comment:
This might return a list of UserGroups if there are multiple groups with the same name